tools/query_verification/src/main/java/com/google/bigquery/QVQuery.java

```diff
@@ -0,0 +1,21 @@
+package com.google.bigquery;
+
+public class QVQuery {
```

Comment:
Just put the full name, QV may be confusing for people, similar for other places. This can also consider Autovalue.